widget/embedded/src/components/Quote/Quote.styles.ts

```diff
@@ -64,6 +64,22 @@ export const stepsDetailsStyles = css({
 export const AllRoutesButton = styled(Button, {
   backgroundColor: 'transparent',
   border: '1px solid $secondary',
+  transition: 'background-color 0.3s ease',
+  paddingLeft: '$10 !important',
+  paddingRight: '$10 !important',
```

Comment:
NIT: I think using inner styles is better because we haven't used `!important `in our codes. However, we should find a better way to override styles later.

Reply:
Fixed